qwp_sender_test.go

```
@@ -0,0 +1,2215 @@
+		t.Fatal("first message should have delta dict flag")
+	}
+
+	// Parse delta dict from first message.
+	off := qwpHeaderSize
+	deltaStart, n, _ := qwpReadVarint(msg1[off:])
+	off += n
+	if deltaStart != 0 {
+		t.Fatalf("msg1 deltaStart = %d, want 0", deltaStart)
+	}
+	deltaCount, n, _ := qwpReadVarint(msg1[off:])
+	off += n
```

golangci-lint (2.11.4)

[error] ineffectual assignment to off

(ineffassign)